src/main/java/gov/cabinetoffice/gap/applybackend/service/GrantMandatoryQuestionService.java

```diff
@@ -173,6 +197,10 @@ public SubmissionQuestion mandatoryQuestionToSubmissionQuestion(final String que
         }
     }
 
+    public boolean doesMandatoryQuestionAlreadyExist(GrantScheme scheme, GrantApplicant applicant) {
```

Comment:
Few suggestions here: 

1.  I don't think we need to pass the full scheme or applicant objects in here. I think we could pass the Scheme ID and applicant ID. In general we should pass as little data around the application as possible and method params should require the smallest number of parameters/smallest data structures as parameters as possible. 
2.  JPA offers a query type of `existsBySomethingAndSomething` which would be more appropriate to use here. [See this article](https://www.baeldung.com/spring-data-exists-query)
3.  In java, the naming convention for a method like this should really be something like `existsBySchemeIdAndApplicantId`